src/views/DogCard/DogList.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import './DogList.css';
+
+import { fetchDogs } from '../../services/fetchdogs';
+
+export default function Home() {
+  const [dog, setDog] = useState([]);
+
+  useEffect(()=> {
+    const fetchDoggosFunct = async () => {
+      const data = await fetchDogs();
+      setDog(data);
+    };
+    fetchDoggosFunct();
+  }, []);
```

Comment:
Don't forget to add in loading state here, perhaps within a try/catch so that we can check for those third party errors.